examples/create_signal/from_tabular_data.py

```diff
@@ -0,0 +1,45 @@
+"""
+Creates a signal1D from tabular data
+====================================
+
+This example creates a signal from tabular data, where the axis are given in values
+(the ``x`` column) and the tabular data are ordered in columns with 5 columns
+containing each 20 values and each columns corresponding to a position in the
+navigation space.
+"""
+
+import numpy as np
+import hyperspy.api as hs
+
+#%%
+# Create a set of tabular data:
+
+x = np.linspace(0, 10, 20)
+y = np.random.default_rng().random((20, 5))
+
+#%%
+# Before creating the signal, define the axes of the signal:
+
+axes = [
+    # lendth of the navigation axis
+    dict(size=y.shape[1], scale=0.1, units="nm"),
```

Comment:
```suggestion
    dict(size=y.shape[1], scale=0.1, name="Position", units="nm"),
```
Define names for axes?